extensions/wikia/PhalanxII/templates/PhalanxSpecial_main.php

```diff
@@ -33,26 +33,32 @@
 						<!-- Format -->
 						<div class="clearfix">
 							<div class="left-spacer">&nbsp;</div>
-							<?= Xml::check( 'wpPhalanxFormatRegex', !empty( $data['regex'] ), array( 'id' => 'wpPhalanxFormatRegex' ) ) ?>
-							<label for="wpPhalanxFormatRegex"><?= wfMessage( 'phalanx-format-regex' )->escaped() ?></label>
+							<label for="wpPhalanxFormatRegex">
+								<input type="checkbox" name="wpPhalanxFormatRegex" value="1" <? if ( !empty( $data['regex'] ) ): ?>checked="checked" <? endif; ?>/>
+								<?= wfMessage( 'phalanx-format-regex' )->escaped() ?>
+							</label>
 
-							<?= Xml::check( 'wpPhalanxFormatCase', !empty( $data['case'] ), array( 'id' => 'wpPhalanxFormatCase' ) ) ?>
-							<label for="wpPhalanxFormatCase"><?= wfMessage( 'phalanx-format-case' )->escaped() ?></label>
+							<label for="wpPhalanxFormatCase">
+								<input type="checkbox" name="wpPhalanxFormatCase" value="1" <? if ( !empty( $data['case'] ) ): ?>checked="checked" <? endif; ?>/>
+								<?= wfMessage( 'phalanx-format-case' )->escaped() ?>
+							</label>
 
-							<?= Xml::check( 'wpPhalanxFormatExact', !empty( $data['exact'] ), array( 'id' => 'wpPhalanxFormatExact' ) ) ?>
-							<label for="wpPhalanxFormatExact"><?= wfMessage( 'phalanx-format-exact' )->escaped() ?></label>
+							<label for="wpPhalanxFormatExact">
+								<input type="checkbox" name="wpPhalanxFormatExact" value="1" <? if ( !empty( $data['exact'] ) ): ?>checked="checked" <? endif; ?>/>
+								<?= wfMessage( 'phalanx-format-exact' )->escaped() ?>
+							</label>
 						</div>
 						<!-- Expiry-->
 						<div class="clearfix">
 							<label>
-								<strong><?= wfMsg( 'phalanx-label-expiry' ) ?></strong>
+								<strong><?= wfMessage( 'phalanx-label-expiry' )->escaped(); ?></strong>
 								<?php if (!empty($editMode)): ?>
 								<span class="expires"><?php
 									if ($data['expire'] === null) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "expire" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)